compute/cker/src/train/Loss.test.cc

```diff
@@ -19,6 +19,85 @@
 #include <gtest/gtest.h>
 #include <vector>
 
+namespace
+{
+using namespace nnfw::cker;
+
+template <typename T> class LossCCEVerifier
+{
+public:
+  LossCCEVerifier(const Shape &in_shape, const Shape &out_shape)
+    : _in_shape{in_shape}, _out_shape{out_shape}
+  {
+  }
+
+  void verifyForward(const std::vector<T> &y_pred, const std::vector<T> &y_true,
+                     const std::vector<T> &expected)
+  {
+    assert(y_pred.size() == y_true.size());
+
+    std::vector<T> output(_out_shape.FlatSize());
+    const int N = _in_shape.Dims(0);
+    const int D = _in_shape.FlatSize() / N;
+
+    nnfw::cker::train::CategoricalCrossEntropy(y_pred.data(), y_true.data(), output.data(), N, D);
+
+    for (int i = 0; i < output.size(); ++i)
+    {
+      EXPECT_FLOAT_EQ(output[i], expected[i]);
```

Comment:
As we use `EXPECT_FLOAT_EQ`, the input/expected/... needs to be selected carefully. What about leaving a comment "Don't be panic when it fails after kernel implementation or input is changed.".